server/server.js

```diff
@@ -176,6 +176,20 @@ app.post('/qa/postAnswer', (req, res) => {
     });
 })
 
+
+/* post request to the api to create cart data */
+app.post('/cart', (request, response) => {
```

Comment:
i'm worried this never sends a response back , does it cause the browser to hang if you Post to this?

Reply:
Good point Phil! I just made the changes